main.cpp

```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun: headphones
+//  action 1: Transduce electric to acoustic energy
+headphones.transduce();
+//  action 2: Cancel Noise
+headphones.cancelNoise();
+//  action 3: Lights up led when battery is low
+headphones.ledOn(); 
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun: Audio Interface
+//  action 1: Powers microphone
+audioInterface.phantomPowerOn();
+//  action 2: converts analog signals to digital
+audioInterface.analogToDigital();
+//  action 3: converts digital signals to analog
+audioInterface.digitalToAnalog();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun: coffee maker
+//  action 1: makes coffee
+coffeeMaker.makeCoffee();
+//  action 2: counts time
+coffeeMaker.countTime();
+//  action 3: beeps when timer is out
+coffeeMaker.beep();
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
-//  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun: Dog 
+//  action 1: Barks
+dog.bark();
+//  action 2: Poops
+dog.poop();
+//  action 3: chases tail
+dog.chaseTail();
+//  5)
+//  Noun: Washing machine
+//  action 1: fills tank with water
+washingMachine.fillTank();
+//  action 2: rotates tank 
+washingMachine.rotateTank();
+//  action 3: dries clothes
+ washingMachine.dryCycle();
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun: Printer
+//  action 1:n Pulls paper
+printer.pullPaper();
+//  action 2: checks ink cartridges
+printer.checkCartridges();
+//  action 3: scans 
+printer.scan();
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun: baby
+//  action 1: eats
+baby.eat();
+//  action 2: smiles
+baby.smile();
+//  action 3: cries
+baby.cry();
 //  8)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
-//  9)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
-//  10)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun: Roomba
+//  action 1: spins brushes
+roomba.spinnBrushes();
+//  action 2: changes direction
+roomba.changeDirection();
+//  action 3: goes to charging dock
+roomba.goToDock();

```

Comment:
As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
**REPLACE** the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.
**Do not** create an 11th noun.